lib/twilio/device.ts

```diff
@@ -1367,6 +1423,15 @@ class Device extends EventEmitter {
     });
   }
 
+  /**
+   * Setup logger's loglevel
+   */
+  private _setupLoglevel(logLevel?: LogLevelDesc): void {
+    const level = typeof logLevel === 'number' ? logLevel : LogLevels.ERROR;
```

Comment:
This limits users to only be able to use numbers, but LogLevelDesc also has strings such as `error` and `debug`.

Reply:
Good catch! This seems like an existing bug (I only moved this logic to its own method). I'll create another ticket to not block this PR. Let me know if you don't agree.